recipes/llvm-core/all/conanfile.py

```diff
@@ -261,11 +348,19 @@ def generate(self):
             "LLVM_ENABLE_FFI": self.options.with_ffi,
             "LLVM_ENABLE_ZLIB": "FORCE_ON" if self.options.with_zlib else False,
             "LLVM_ENABLE_LIBXML2": "FORCE_ON" if self.options.with_xml2 else False,
-            "LLVM_ENABLE_TERMINFO": self.options.with_terminfo
         }
+        if Version(self.version) < 19:
+            cmake_variables["LLVM_ENABLE_TERMINFO"] = self.options.get_safe("with_terminfo")
+        else:
+            cmake_variables["LLVM_ENABLE_ZSTD"] = "FORCE_ON" if self.options.get_safe("with_zstd") else False
+
         if self.options.targets != "all":
             cmake_variables["LLVM_TARGETS_TO_BUILD"] = self.options.targets
 
+        if is_msvc(self):
+            # https://discourse.llvm.org/t/failing-to-compile-llvm-on-windows-with-clang
+            cmake_variables["MSVC"] = True
+
```

Comment:
```suggestion
```

I would not enforce this native cmake variable. Using clang on Windows has several flavours, msvc is only one case: https://blog.conan.io/2022/10/13/Different-flavors-Clang-compiler-Windows.html

Reply:
Removed. This was fixed in https://github.com/llvm/llvm-project/pull/95505